src/Http/Http.Extensions/gen/RequestDelegateGeneratorSources.cs

```diff
@@ -95,6 +113,14 @@ private static Task WriteToResponseAsync<T>(T? value, HttpContext httpContext, J
         }
 """;
 
+    public static string PopulateEndpointMetadata => """
```

Comment:
What's the difference between this and `PopulateEndpointParameterMetadataMethod`?

Reply:
Nothing - was just a skeleton left over from some earlier hacking around.